Updates test skips & RuboCop

MSP-Greg opened the pull request:
Updates helper.rb skip methods & RuboCop to Ruby 2.2

olleolleolle commented:
The new skip methods improve the legibility of the tests! The RuboCop update seems to have worked nicely.

MSP-Greg commented:
I just rebased and changed the PR to allow showing the real location of the skip.  Previous tests and the original commit would show the location of the skip as somewhere in helper.rb.

The update fixes that...

And I had a typo (I thought I changed something from an instance variable to a local, but I forgot one `@`).  Updated twice...
